sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2081,13 +2083,13 @@ public void onCaseChanged(CaseDataDto existingCase, Case newCase, boolean syncSh
 						newCase.setCaseClassification(classification);
 						newCase.setClassificationUser(null);
 						newCase.setClassificationDate(new Date());
+						setClassificationInfo = false;
 					}
 				}
-			} else if (newCase.getCaseClassification() != existingCase.getCaseClassification()) {
-				newCase.setClassificationUser(userService.getCurrentUser());
-				newCase.setClassificationDate(new Date());
 			}
-		} else if (newCase.getCaseClassification() != existingCase.getCaseClassification()) {
+		}
+
+		if (setClassificationInfo && existingCase != null && newCase.getCaseClassification() != existingCase.getCaseClassification()) {
```

Comment:
when a new case is created (maybe from the rest api) and the classification is already filled shouldn't these fields also be filled?